support/nginx-default.conf

```diff
@@ -6,26 +6,10 @@ server {
   gzip_vary on;
   gzip_min_length 10240;
   gzip_proxied expired no-cache no-store private auth;
-  gzip_types text/plain text/css text/xml text/html text/javascript application/x-javascript application/xml;
+  gzip_types text/plain text/css text/xml text/javascript application/x-javascript application/xml;
```

Comment:
gotta keep text/html in the gzip section here.

Reply:
I was getting a warning so I removed it. It's included as a gzip type by default. 
http://nginx.org/en/docs/http/ngx_http_gzip_module.html

Reply:
oh! ok cool beans then.